code/ship/ship.cpp

```diff
@@ -2317,6 +2318,11 @@ int parse_ship_values(ship_info* sip, bool first_time, bool replace)
 		if(optional_string("+Auto Spread:")) {
 			stuff_float(&sip->auto_shield_spread);
 		}
+		if(optional_string("+Minimum Weapon Span:")) {
+			stuff_float(&sip->auto_shield_spread_min_span);
+		} else if (first_time) {
+			sip->auto_shield_spread_min_span = sip->auto_shield_spread;
```

Comment:
If this only gets set automatically the first time, a modular table that adjusts the auto shield spread of a ship class won't update the minimum weapon span; this would be a behavior difference between FSO versions.

The most modder-friendly way to do it would be to make it so that as long as Minimum Weapon Span is never explicitly set, it's always automatically the same as the value of Auto Spread. Two ways to do this; either change the checks in collideshipweapon to use auto_shield_spread_min_span only if it's not -1 (and auto_shield_spread otherwise), or set auto_shield_spread_min_span to the value of auto_shield_spread (if auto_shield_spread_min_span is -1) after parsing all modular ship tables is done (the performance savings of not doing it at runtime is probably negligable, though).